app/components/display/taxonomy_card_component/taxonomy_card_component.html.haml

```diff
@@ -0,0 +1,25 @@
+.taxonomy-card{'data-controller': 'reveal-component'}
+  .title-bar
+    .title
+      = "#{@taxonomy.name} (#{@taxonomy.acronym})"
+    %a{href: "https://#{@taxonomy.acronym}.#{$UI_URL.sub("https://", "")}"}
+      = inline_svg_tag('icons/slices.svg', class: "taxonomy-slice-svg #{@taxonomy.is_slice ? '' : 'd-none'}")
+  %a.ontologies{href: "/ontologies?#{@taxonomy.id.split('/')[-2]}=#{@taxonomy.acronym}"}
+    = inline_svg_tag('icons/ontology.svg')
+    .number-of-ontologies
+      = "#{@taxonomy.ontologies.length} ontologies"
+  .description
```

Comment:
need to use the see more component here

Reply:
this one 
![image](https://github.com/user-attachments/assets/30585656-e6f2-480a-bc03-9d1d6dab38a7)
it will handle the show more/less automatically